projects/ada4250_ardz/src.mk

```diff
@@ -3,19 +3,28 @@
 SRC_DIRS += $(PROJECT)/src
 SRC_DIRS += $(DRIVERS)/amplifiers/ada4250
 SRC_DIRS += $(NO-OS)/iio/iio_app
-SRC_DIRS += $(NO-OS)/drivers/irq
-SRC_DIRS += $(NO-OS)/drivers/gpio
 
-SRC_DIRS += $(PLATFORM_DRIVERS)
```

Comment:
what was wrong with this line?

Reply:
As I mentioned in the other comment, I didn't want to add the whole folder (which includes also unused files).